src/main/java/net/fabricmc/example/mixin/ExampleMixin.java

```diff
@@ -1,15 +1,20 @@
 package net.fabricmc.example.mixin;
 
+import net.minecraft.client.gui.screen.Screen;
 import net.minecraft.client.gui.screen.TitleScreen;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
 @Mixin(TitleScreen.class)
-public class ExampleMixin {
+public class ExampleMixin extends Screen {
 	@Inject(at = @At("HEAD"), method = "init()V")
 	private void init(CallbackInfo info) {
 		System.out.println("This line is printed by an example mod mixin!");
 	}
+	@Inject(method = "render(IIF)V", at = @At("RETURN"))
+	public void render(int mouseX, int mouseY, float delta, CallbackInfo info) {
```

Comment:
Method should also be private

Reply:
Haven't gotten all the way through the Welcome To Mixins, but curious as to why they should be private?

Reply:
because they can be and have no reason to be public (though in sponge mixin wiki the showcase have them public)